doc/095_considerations.rst

```diff
@@ -0,0 +1,36 @@
+..
+  Normally, there are no heading levels assigned to certain characters as the structure is
+  determined from the succession of headings. However, this convention is used in Python’s
+  Style Guide for documenting which you may follow:
+
+  # with overline, for parts
+  * for chapters
+  = for sections
+  - for subsections
+  ^ for subsubsections
+  " for paragraphs
```

Comment:
I think these comments could be removed...

Reply:
Probably.  I just threw them in there for my reference, never removed them.